src/services/player.ts

```diff
@@ -379,12 +379,14 @@ export default class {
     this.queue = [];
   }
 
-  move(from: number, to: number): void {
+  move(from: number, to: number): string {
     if (from > this.queueSize() || to > this.queueSize()) {
       throw new Error('Move index is outside the range of the queue.');
     }
 
     this.queue.splice(this.queuePosition + to, 0, this.queue.splice(this.queuePosition + from, 1)[0]);
+
+    return this.queue[this.queuePosition + to].title;
```

Comment:
Let's have this return the entire queued track instead (drop the `.title`) and then callers of this method can select what they want.

Reply:
done as requested